spring-beans/src/main/java/org/springframework/beans/support/PropertyComparator.java

```diff
@@ -115,8 +113,9 @@ private Object getPropertyValue(Object obj) {
 		// (similar to JSTL EL). If the property doesn't exist in the
 		// first place, let the exception through.
 		try {
-			this.beanWrapper.setWrappedInstance(obj);
-			return this.beanWrapper.getPropertyValue(this.sortDefinition.getProperty());
+            BeanWrapperImpl beanWrapper = new BeanWrapperImpl(false);
+            beanWrapper.setWrappedInstance(obj);
+            return beanWrapper.getPropertyValue(this.sortDefinition.getProperty());
 		}
```

Comment:
Please note that the formatting you introduced fails the build with the following.

```
> Task :spring-beans:checkstyleMain
[ant:checkstyle] [ERROR] /.../spring-beans/src/main/java/org/springframework/beans/support/PropertyComparator.java:116: Line has leading space characters; indentation should be performed with tabs only. [RegexpSinglelineJava]
[ant:checkstyle] [ERROR] /.../spring-beans/src/main/java/org/springframework/beans/support/PropertyComparator.java:117: Line has leading space characters; indentation should be performed with tabs only. [RegexpSinglelineJava]
[ant:checkstyle] [ERROR] /.../spring-beans/src/main/java/org/springframework/beans/support/PropertyComparator.java:118: Line has leading space characters; indentation should be performed with tabs only. [RegexpSinglelineJava]
```

Please make sure you run `./gradlew check` before submitting a PR to catch such errors locally.

I'll fix the formatting before merging, so there's no need to update this PR.